templates/include/address-space-definitions.yaml

```diff
@@ -63,8 +63,125 @@ data:
       kind: ConfigMap
       metadata:
         name: broker-prometheus-config
-    - apiVersion: extensions/v1beta1
-      kind: Deployment
+    - apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: qdrouterd-config
+      data:
+        qdrouterd.conf: |-
+          router {
+            mode: interior
+            id: ${HOSTNAME}
+            defaultDistribution: unavailable
+            allowResumableLinkRoute: ${ALLOW_DURABLE_SUBSCRIPTIONS}
```

Comment:
Would be nice if worker threads were still configurable. Also, ideally the image should auto tune it based on cores in the system (or does it already do that?)

Reply:
They are still configurable through the configmap. There was no template parameter for them previously, so they would only have been configurable then be directly editing the deployment for the router.